src/main/java/uk/gov/pay/products/model/Payment.java

```diff
@@ -34,6 +35,9 @@
     private Long amount;
     @JsonProperty(FIELD_GOVUK_STATUS)
     private String govUkStatus;
+    @JsonIgnore
+    private Integer gatewayAccountId;
```

Comment:
I probably would not include this in the Payment object at all. It can only remain in the Entity Object